.fixtures.yml

```diff
@@ -3,4 +3,9 @@
 ---
 fixtures:
   forge_modules:
```

Comment:
please don't link to the forge. we try to use the master/main branch of modules during unit tests. the layout would be:
```yaml
---
fixtures:
  repositories:
    concat: https://github.com/puppetlabs/puppetlabs-concat.git
    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
    ...
```

Reply:
the idea is that CI runs way more often than people release their modules. testing against git ensures that we catch breaking changes before they are released and we can adjust our code and/or inform upstream. Often breaking changes are unintentional. 
In acceptance tests on the other hand we work with official forge releases to check everything works.